apps/dashboard/app/models/project.rb

```diff
@@ -8,21 +8,17 @@ class Project
   include IconWithUri
   extend JobLogger
 
-  def self.from_directory(dir)
+  def self.from_directory!(dir)
```

Comment:
You have a bug here where you changed this signature but still call the regular version.

Reply:
My bad let me fix it.